specification/iotcentral/resource-manager/Microsoft.IoTCentral/preview/2021-11-01-preview/iotcentral.json

```diff
@@ -0,0 +1,1259 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "AppProperties": {
+      "description": "The properties of an IoT Central application.",
+      "type": "object",
+      "properties": {
+        "applicationId": {
+          "description": "The ID of the application.",
+          "type": "string",
+          "readOnly": true
+        },
+        "displayName": {
+          "description": "The display name of the application.",
+          "type": "string"
+        },
+        "subdomain": {
+          "description": "The subdomain of the application.",
+          "type": "string"
+        },
+        "template": {
+          "description": "The ID of the application template, which is a blueprint that defines the characteristics and behaviors of an application. Optional; if not specified, defaults to a blank blueprint and allows the application to be defined from scratch.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "state": {
+          "description": "The current state of the application.",
+          "$ref": "#/definitions/AppState"
+        },
+        "publicNetworkAccess": {
+          "description": "Whether requests from the public network are allowed.",
+          "$ref": "#/definitions/PublicNetworkAccess"
+        },
+        "networkRuleSets": {
+          "description": "Network Rule Set Properties of this Iot Central application.",
+          "$ref": "#/definitions/NetworkRuleSets"
+        },
+        "privateEndpointConnections": {
+          "description": "Private endpoint connections created on this Iot Central application.",
+          "type": "array",
+          "items": {
+            "$ref": "../../../../../common-types/resource-management/v4/privatelinks.json#/definitions/PrivateEndpointConnection"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "PublicNetworkAccess": {
+      "description": "Whether requests from the public network are allowed.",
+      "type": "string",
+      "enum": [
+        "Enabled",
+        "Disabled"
+      ],
+      "x-ms-enum": {
+        "name": "PublicNetworkAccess",
+        "modelAsString": true
+      }
+    },
+    "NetworkRuleSets": {
+      "description": "Network Rule Set Properties of this Iot Central application.",
+      "type": "object",
+      "properties": {
+        "applyToDevices": {
```

Comment:
Synced with PM, can these be updated to:

applyToIoTCentral : "Whether these rules apply for connectivity via IoT Central web portal and APIs"

applyToDevices: "Whether these rules apply for device connectivity to IoT Hub and Device Provisioning service associated with this application"